libr/include/r_anal.h

```diff
@@ -744,11 +724,11 @@ typedef struct r_anal_bind_t {
 
 typedef const char *(*RAnalLabelAt) (RAnal *a, RAnalFunction *fcn, ut64);
 
-#define R_ANAL_VAR_KIND_ANY 0
-#define R_ANAL_VAR_KIND_ARG 'a'
-#define R_ANAL_VAR_KIND_REG 'r'
-#define R_ANAL_VAR_KIND_BPV 'b'
-#define R_ANAL_VAR_KIND_SPV 's'
+typedef enum {
+	R_ANAL_VAR_KIND_REG = 'r',
```

Comment:
so u keep a list of args and list of vars? lgtm

Reply:
For now it is just one RPVector per function storing all vars. We can optimize later for the kinds of accesses we need if necessary after profiling.